runtime/src/accounts_index.rs

```diff
@@ -1302,11 +1302,12 @@ impl<T: IndexValue> AccountsIndex<T> {
         //  pubkey looked up
         //  refcount of entry in index
         F: FnMut(bool, &SlotList<T>, &Pubkey, RefCount) -> bool,
+        I: Iterator<Item = &'a Pubkey>,
```

Comment:
I tend to use `IntoIterator` as per the [Rust API guidelines](https://rust-lang.github.io/api-guidelines/flexibility.html#functions-minimize-assumptions-about-parameters-by-using-generics-c-generic).

Reply:
done